test/e2e/features/pageObjects/common/CaseManager.js

```diff
@@ -46,10 +47,19 @@ class CaseManager {
     }
 
     async clickPreviousButton(){
-        await BrowserWaits.waitForElement(this.previousBtn); 
+        await BrowserWaits.waitForElement(this.previousBtn);
         var thisPageUrl = await browser.getCurrentUrl();
         await this.previousBtn.click();
-        await BrowserWaits.waitForPageNavigation(thisPageUrl); 
+        await BrowserWaits.waitForPageNavigation(thisPageUrl);
+    }
+
+    async submitComment(){
```

Comment:
as this function is clicking submit button in page irrespective of event and can be used in any event page to submit, can you update this something like "clickSubmit" so it can be reused.

Reply:
done